File: Front-end/src/app/DTO/MailDTO.ts
export class MailDTO{
    mailId: string = "";
    folderId: string = "";
    from: string = "";
    to: string = "";
    subject: string = "";
    content: string = "";
    timestamp: string = "";
    state: string = 'mail';
    isStarred: boolean = false;
    priority: number = 4;

}

File: Back-end/src/main/java/com/accursed/mailserver/dtos/MailDTO.java
package com.accursed.mailserver.dtos;

import com.accursed.mailserver.models.Attachment;
import org.springframework.web.multipart.MultipartFile;

import java.io.File;
import java.sql.Timestamp;
import java.util.Set;

public class MailDTO {
    public String mailId;
    public String folderId;
    public String userId;
    public String id;
    public String from;
    public String to;
    public String subject;
    public String content;
    public Timestamp timestamp;
    public String state;
    public Boolean isStarred;
    public Integer priority;
    public Set<Attachment> attachments;
    public Set<MultipartFile> files;

}
